hazelcast/src/main/java/com/hazelcast/config/NearCacheConfigReadOnly.java

```diff
@@ -81,4 +81,9 @@ public NearCacheConfig setLocalUpdatePolicy(LocalUpdatePolicy localUpdatePolicy)
     public NearCacheConfig setEvictionConfig(EvictionConfig evictionConfig) {
         throw new UnsupportedOperationException("This config is read-only");
     }
+
+    @Override
+    public EvictionConfig getEvictionConfig() {
+        return super.getEvictionConfig().getAsReadOnly();
```

Comment:
This broke the EE build before, because it triggered a check of the `EvictionConfig`, which failed in the context of some Near Cache tests.